tests/kernel/wave/attention/extend_attention_test.py

```diff
@@ -140,18 +140,20 @@ def create_inputs(
     H_KV = shape.num_kv_heads
     H_Q = shape.num_query_heads
     D = shape.head_size
-
-    b_seq_len_prefix = torch.randint(
-        1, N_CTX // 2, (B,), dtype=torch.int32, device="cuda"
-    )
-    # TODO: Make this work for values != 64.
-    b_seq_len_prefix[0] = 64
-    b_seq_len_prefix[1] = 64
-    b_seq_len_extend = torch.randint(
-        1, N_CTX // 2, (B,), dtype=torch.int32, device="cuda"
-    )
-    b_seq_len_extend[0] = 128
-    b_seq_len_extend[1] = 256
+    # TODO: Enable when we have proper masking for attention.
+    # b_seq_len_prefix = torch.randint(
+    #     1, N_CTX // 2, (B,), dtype=torch.int32, device="cuda"
+    # )
+    b_seq_len_prefix = torch.empty((B,), dtype=torch.int32, device="cuda")
+    for i in range(B):
+        b_seq_len_prefix[i] = shape.block_size * (i + 1)
```

Comment:
Wouldn't the values here get overriden by L156
```
b_seq_len_prefix[i] = shape.block_size * (i + 2)
```

Reply:
yes that was a typo. will fix.